src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,32 @@
 import React from 'react';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  name: string;
+  email: string;
+  body: string
+};
+
+export const CommentInfo: React.FC<Props> = ({
+  name, email, body,
+}) => (
+  <>
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{name}</strong>
+
+        {' by '}
+
+        <a
+          className="CommentInfo__email"
+          href="mailto:Telly_Lynch@karl.co.uk"
```

Comment:
I'm not sure that all of the email links should lead to one address. Fix it, please